### DISCUSSION TITLE
Issues

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#37519] [Projects and Issues] [answered]
Hey everyone,

So basically, I have added issues in some of my repos so it means that we have to make the desired changes in our code right? Or something else has to be done.

**Comment by airtower-luna** [comment, marked as answer]
Technically, it's up to you (or in general the repository owner) what to do with issues. :grin: 

But yeah, generally the idea is that you either make changes/fixes to resolve the issues, or close them if you don't intend to. Though it's not unusual for issues to stay open a long time, e.g. if the issue describes a good idea but there isn't time to implement it. Or maybe the maintainer of a software is open to a certain change *if someone provides a patch* but not interested in implementing it themselves, and leaves the issue open in case someone wants to do that.

If you have a PR with changes that resolve an issue it's usually useful to put a line like `Closes: #1234` into the description. That will close the issue as soon as the PR gets merged.

**Comment by Subash1811** [comment]
I can't collaborate contribution in my teams because I can't accept collaboration invitation i want to join my contribution team .. anyone know the rectify solution the problem.. plse tell me..
